jbpm-workitems/jbpm-workitems-webservice/src/test/java/org/jbpm/process/workitem/webservice/WebServiceWorkItemHandlerTest.java

```diff
@@ -235,6 +235,50 @@ public void testExecuteSyncOperationWithBasicAuth() throws Exception {
                      authorizationPolicy.getPassword());
     }
     
+    @Test
+    public void testExecuteSyncOperationWithBasicAuthWithParameter() throws Exception {
```

Comment:
You can remove this as there is no checked exception thrown by this test method
```suggestion
    public void testExecuteSyncOperationWithBasicAuthWithParameter() {
```